api/views/maps.py

```diff
@@ -0,0 +1,86 @@
+from api import app
+from flask import Blueprint, request
+from api import db
+from api.models import Map
+import json
+from flask import jsonify
+#from api.utils import InvalidUsage
+from sqlalchemy import func
+# import time
+# from datetime import date
+# import uuid
+from flask_sqlalchemy import SQLAlchemy
+
+
+mod = Blueprint('maps', __name__)
+
+def create_response(data={}, status=200, message=''):
+    """
+    Wraps response in a consistent format throughout the API
+    Format inspired by https://medium.com/@shazow/how-i-design-json-api-responses-71900f00f2db
+    Modifications included:
+    - make success a boolean since there's only 2 values
+    - make message a single string since we will only use one message per response
+    IMPORTANT: data must be a dictionary where:
+    - the key is the name of the type of data
+    - the value is the data itself
+    """
+    response = {
+        'success': 200 <= status < 300,
+        'code': status,
+        'message': message,
+        'result': data
+    }
+    return jsonify(response), status
+#none of these have been comprehsnsively tested yet
+
+@app.route('/maps', methods = ['GET', 'POST'])
+# Get all map years.
+# get function is definitely not functional
+def map_():
+    if request.method == 'GET':
+        print(Map.query.all())
+        data_in = Map.query.all()
+        data_in = list(filter(lambda x : x.map_year, data_in))
```

Comment:
there exists a `to_dict()` on each model that transforms it from the db representation to a python dictionary. you can then use that create your response. 

also, make sure to follow the `create_response`'s `data` parameter format: https://github.com/hack4impact-uiuc/wiki/wiki/Our-REST-API-Specification#response-json-format